lib/avirwrapper.php

```diff
@@ -67,6 +67,7 @@ public function fopen($path, $mode){
 		if (is_resource($stream)
 			&& $this->isWritingMode($mode)
 			&& $this->isScannableSize(basename($path))
+			&& strpos($path, 'uploads/') !== 0
```

Comment:
can I trick this by having a path "myfolder/something/uploads/infected.exe" ?

Reply:
or is this check very subtle and means: only go there either when "uploads" is not at the start or when "uploads" is not found at all ? A PHP comment would be appreciated to clarify this implicit confusing logic

Reply:
@PVince81 I think `strpos($path, 'uploads/') !== 0` is self-descriptive as it reads as `path does not start with`